utils/src/utils/dateUtils.ts

```diff
@@ -0,0 +1,14 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+export namespace dateUtils {
+    export function getFormattedDurationInMinutesAndSeconds(start: Date, end: Date): string {
```

Comment:
The `dayjs` package (which we already use) can probably do this for us.

Reply:
Let me know if this is what you had in mind or if you were preferring that I remove the `dateUtils` entirely!

Reply:
This is what I was thinking: https://day.js.org/docs/en/durations/format